internal/job/testing.go

```diff
@@ -0,0 +1,74 @@
+package job
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/boundary/internal/db"
+	"github.com/hashicorp/boundary/internal/db/timestamp"
+	"github.com/hashicorp/boundary/internal/kms"
+	"github.com/hashicorp/boundary/internal/servers"
+	wrapping "github.com/hashicorp/go-kms-wrapping"
+	"github.com/hashicorp/go-uuid"
+	"github.com/jinzhu/gorm"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+var (
+	zeroTime   = &timestamp.Timestamp{Timestamp: &timestamppb.Timestamp{}}
```

Comment:
perhaps name these package vars with a prefix of `test`?

Reply:
ty